lib/sensu/server/handle.rb

```diff
@@ -126,12 +126,21 @@ def transport_handler(handler, event_data)
       # @param event_id [String] event UUID
       def handler_extension(handler, event_data, event_id)
         handler.safe_run(event_data) do |output, status|
-          @logger.info("handler extension output", {
-            :extension => handler.definition,
-            :event => { :id => event_id },
-            :output => output,
-            :status => status
-          })
+          if output == '' and status == 0
```

Comment:
How about this instead?

```
log_level = (output == "" && status == 0 ? :debug : :info 
@logger.send(log_level, "handler extension output", {
...
})
```

Reply:
This pattern is already used elsewhere, https://github.com/sensu/sensu/blob/9059bf10125221cd4806cc0ad6758f43fe216e8b/lib/sensu/server/process.rb#L258-L259

Reply:
Only slight tweaks to make it more ruby:
```ruby
log_level = (output.empty? && status.zero?) ? :debug : :info 
@logger.send(log_level, "handler extension output", {
...
})
```

Reply:
ok I have modified the pull request...